JSDemos/Demos/DataGrid/StatePersistence/description.md

```diff
@@ -1 +1,5 @@
-Sort and filter data, group, reorder and resize columns, change page numbers and page size. Once you are done, refresh the web page to&nbsp;see that the grid&rsquo;s state is&nbsp;automatically persisted to&nbsp;continue working from where you stopped.
\ No newline at end of file
+**DataGrid** state can be persistent - if a user sorts and filters data, groups, reorders and resizes columns or makes other changes, then the widget saves these modifications and restores them when a user reloads the page. The full list of modifications that can be saved is in the [stateStoring](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/) documentation.
+
+To enable state persistence, set **stateStoring**.[enabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#enabled) to **true** and specify the [storageKey](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#storageKey) and [type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#type) options. Depending on the type, the state can be saved to a [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) or [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage). With localStorage, the state persists across browser sessions; with sessionStorage, it is reset after the current session. This demo uses localStorage.
```

Comment:
When you refer to the third-party resources, use HTML links, not markdown. This rule exists because markdown does not allow you to open link in a new tab, and the latter is a required condition. So it should be 
`<a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage" target="_blank">localStorage</a>`. The same applies to the sessionStorage link.

I would also make these names bold when they are not links.

Reply:
Thank you. Applied these edits.